db/tasks.php

```diff
@@ -24,50 +24,68 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$tasks = array(
-    array(
+$tasks = [
+    [
         'classname' => 'mod_zoom\task\update_meetings',
         'blocking' => 0,
         'minute' => '0',
         'hour' => '0',
         'day' => '*',
         'month' => '*',
         'dayofweek' => '*'
-    ),
-    array(
+    ],
+    [
         'classname' => 'mod_zoom\task\get_meeting_reports',
         'blocking' => 0,
         'minute' => '0',
         'hour' => '0,12',
         'day' => '*',
         'dayofweek' => '*',
         'month' => '*'
-    ),
-    array(
+    ],
+    [
         'classname' => 'mod_zoom\task\reset_api_calls',
         'blocking' => 0,
         'minute' => '59',
         'hour' => '23',
         'day' => '*',
         'dayofweek' => '*',
         'month' => '*'
-    ),
-    array(
+    ],
+    [
         'classname' => 'mod_zoom\task\send_notifications',
         'blocking' => 0,
         'minute' => '*/15',
         'hour' => '*',
         'day' => '*',
         'dayofweek' => '*',
         'month' => '*'
-    ),
-    array(
+    ],
+    [
         'classname' => 'mod_zoom\task\insert_recordings',
         'blocking' => 0,
         'minute' => '*/30',
         'hour' => '*',
         'day' => '*',
         'dayofweek' => '*',
         'month' => '*'
-    )
-);
+    ],
+    [
+        'classname' => 'mod_zoom\task\add_meeting_registrant',
+        'blocking' => 0,
+        'minute' => '*',
```

Comment:
@Ash-5 not every minute please

Reply:
@meyyappanv once you say things are fine no changes are needed I will change the cron run time to every 10min as it runs in ultron.